src/components/work/AuthenticationTestimonials/AuthenticationTestimonials.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import { Carousel, CarouselItem, CarouselIndicators, CarouselCaption, Col, Container } from 'reactstrap';
+
+
+// should we create a custom wrapper over this carousel?  a little hard to use
+const items = [
+  {
+    altText: 'This is the first testimonial',
+    caption: 'Slide 1',
+  },
+  {
+    altText: 'This could be the second testimonial',
+    caption: 'Slide 2',
+  },
+  {
+    altText: 'Sometimes this is the third testimonial',
+    caption: 'Slide 3',
+  },
+];
+
+class AuthenticationTestimonials extends React.Component {
```

Comment:
can we shorten the prefix to `Auth` instead of `Authentication`? It has the same meaning and a lot shorter to type.

Reply:
👍